.pre-commit-config.yaml

```diff
@@ -1,35 +1,3 @@
-repos:
```

Comment:
^ might wanna keep this 😄

Reply:
yeah just realized and fixed it. It still worked locally!!!